lib/masamune/transform/postgres/snapshot_dimension.psql.erb

```diff
@@ -28,11 +28,11 @@ http://www.postgresql.org/message-id/CAGnEbohhKmW55oB0FpQd3naXBkwi74E=8DRZBGFjS-
 WITH ranges AS (
   SELECT *,
   CASE WHEN delta = 0
-  THEN 1 ELSE NULL END r
+  THEN 1 ELSE 0 END r
   FROM <%= source.name %>
 ), windows AS (
   SELECT *,
-  SUM(r) OVER (ORDER BY <%= target.window("start_at #{order}", 'delta', 'source_uuid DESC').join(', ') %>) window_id
+  SUM(r) OVER (PARTITION BY <%= target.window.join(', ') %> ORDER BY <%= ["start_at #{order}", 'delta', "source_uuid #{order}"].join(', ') %>) AS window_id
```

Comment:
complete ordering works if all delta records happen after a complete record, but ideally the `window_id` should be computed independently for each `PARTITION `